Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -153,6 +153,11 @@ public void Validate()
                 }
             }
 
+            if (Timeout == 0)
+            {
+                throw new EasyNetQException("Invalid connection string. 'timeout' value must be positive value");
```

Comment:
Does this require to bump version to 5.0.0?

Reply:
API isn't changed(I mean compilation will not break), only small incompatibility which will break a startup of an application.

Reply:
I know, that API did not change but behavior really changed. Anyway, I'm ok with any next version.

Reply:
@sungam3r is right, it should be seen as a breaking change, because the same code will not work anymore, so I will push it in a breaking change version, or try to manage the 0, sorry :(

Reply:
Could we also remove timeout strategy and timeout attribute then? (:

Reply:
Why so? It will be breaking change too. These classes seem to be useful. I seem to even use timeout strategy.

Reply:
Why so? It will be breaking change too. These classes seem to be useful. I seem to even use timeout strategy. I can propose to return to work on #894 and bump major with both changes if you don't want to bump only because timeout :).